src/components/Resource/ResourceDetailsUpdate.tsx

```diff
@@ -100,15 +112,39 @@ export const ResourceDetailsUpdate = (props: resourceProps) => {
   const validateForm = () => {
     const errors = { ...initError };
     let isInvalidForm = false;
+
+    const validators = {
+      facilityValidator: (field: string) =>
```

Comment:
While this works, it isn't the best option readability wise (esp if we add any new fields). Let's switch to case statements as we have in other components.

Also allows to add any extra checks in the future; if we need different error messages.